### DISCUSSION TITLE
Source code

### DISCUSSION THREAD

**Discussion by Romaricbah** [community/community#144538] [Programming Help]
### Body

I need a source code for a professional mern project 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by sasmithaK** [comment]
lugenx/ecohabit 

check out this project. I found it really helpful.

**Comment by denis23x** [comment]
Hi @Romaricbah 

Every professional is different, it is impossible to choose an absolute 🤷‍♂️

You can find many good examples here https://codebase.show/projects/realworld?category=fullstack the same application is written in different ways using different technologies.
